chat-commands.js

```diff
@@ -756,6 +756,12 @@ exports.commands = {
 			}
 		}
 
+		if (targetRoom.subRooms && targetRoom.subRooms.size) {
+			targetRoom.subRooms.forEach(function (room) { room.parent = null; });
+		} else if (targetRoom.parent && targetRoom.parent.subRooms) {
```

Comment:
This should not be an `else`; there's no reason to believe both might not be true.